docs/release_notes/v1.13.0.md

```diff
@@ -0,0 +1,519 @@
+See additional info [here](https://v1-13.docs.dapr.io/reference/arguments-annotations-overview/).
+
+### Standardized error codes
+
+The Dapr PubSub and State APIs now have the appropriate and standardized error codes returned to applications including enriched error details based on gRPC's richer error model.
+The remaining APIs are a work in progress, and community contributions towards this effort are greatly appreciated and very impactful.
+See additional info on the error codes [here](https://docs.dapr.io/reference/api/error_codes/).
+Check the docs for each SDK for per-SDK error code parsing and handling.
+See an [example of the Go SDK error parsing and handling here](https://v1-13.docs.dapr.io/developing-applications/sdks/go/go-client/#error-handling).
+
+### Actor reminders performance improvement
+
+You can now opt-in to make actor reminders data use protobuf serialization instead of JSON, which increases throughput, reduces latency, and improves stability when multiple Dapr instances are operating on the same reminders.
+With this feature enabled, we have observed improvements as high as 40% in Actor Reminders and Workflow benchmarks.
+This serialization method will become default in v1.14.
+
+> **Important:** Once you enable this feature, you should not downgrade the Dapr control plane to an earlier version, as your reminders data may become unreadable.
+
+To enable protobuf serialization for actor reminders, set the following Helm argument on Kubernetes: `dapr_placement.maxActorApiLevel=20`.
+On self-hosted mode, run daprd with the flag: `--max-api-level=20`
+
+### Rust SDK support for actors (alpha)
+
+You can now use Rust to run Dapr Actors, a programming model for highly scalable stateful applications.
+See the [Rust SDK for more details](https://github.com/dapr/rust-sdk/tree/master/examples/actors)
+
+### Components
+
+#### Local name resolver based on SQLite
+
+You can now use a SQLite based name resolver for [service invocation in self-hosted mode](https://docs.dapr.io/developing-applications/building-blocks/service-invocation/service-invocation-overview/).
+This is useful for dealing with corporate firewalls and VPNs that filter mDNS.
+See additional info [here](https://v1-13.docs.dapr.io/reference/components-reference/supported-name-resolution/nr-sqlite/).
+
+#### PostgreSQL state store v2
+
+PostgreSQL has a new v2 implementation which contains improvements to performance and reliability.
+New applications are encouraged to use v2.
+The v1 implementation remains supported and is not deprecated.
+There is no migration path from v1 to v2.
+See additional info [here](https://v1-13.docs.dapr.io/reference/components-reference/supported-state-stores/setup-postgresql-v2/).
+
+#### Azure Blob Storage state store v2
+
+Azure Blob Storage has a new v2 implementation, which is recommended for all new projects as it fixes a backwards-incompatible bug with key prefixes.
```

Comment:
We should file a separate issue to upgrade docs and samples to lead with using these by default.

Reply:
Docs have been updated, do we use ABS somewhere in samples that you're aware of?

Reply:
Now that I look at it I see we don't have a doc on state management in Azure (which would use blobs etc), so I think we're good based on docs.  thank you!